firestore/FirestoreSwiftUIExample/Model/Restaurant.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  Restaurant.swift
+//  FirestoreSwiftUIExample
+//
+
+import Firebase
+import FirebaseFirestoreSwift
+
+struct Restaurant: Identifiable, Codable {
+  var id: String = UUID().uuidString
+
+  var name: String
+  var category: String // Could become an enum
+  var city: String
+  var price: Int // from 1-3; could also be an enum
+  var ratingCount: Int // numRatings
+  var averageRating: Float
+  var photo: URL
+
+  enum CodingKeys: String, CodingKey {
+    case name
+    case category
+    case city
+    case price
+    case ratingCount = "numRatings"
+    case averageRating = "avgRating"
+    case photo
+  }
+}
+
+extension Restaurant {
+
```

Comment:
In some structs, you leave a blank line, it others you don't. Would recommend choosing one approach and then implement consistently.